File: JAVA/hargabarang.java
import java.util.Scanner;

public class hargabarang {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Masukkan nama barang: ");
        String nama = input.nextLine();

        System.out.print("Masukkan harga barang: ");
        double harga = input.nextDouble();

        System.out.print("Masukkan jumlah beli: ");
        int jumlah = input.nextInt();

        double total = harga * jumlah;
        double diskon = 0;

        if (total > 26.100) { 
            diskon = total * 0.010;
        }

        double totalBayar = total - diskon;

        System.out.println("\nSTRUK PEMBELIAN");
        System.out.println("Nama Barang: " + nama);
        System.out.println("Harga Satuan: " + harga);
        System.out.println("Jumlah Beli: " + jumlah);
        System.out.println("Total Harga: " + total);
        System.out.println("Diskon: " + diskon);
        System.out.println("Total Bayar: " + totalBayar);
        System.out.println("SELESAI");

        input.close();
    }
}

File: C++/hargabarang.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string nama;
    float harga;
    int jumlah;

    cout << "Masukkan nama barang: ";
    getline(cin, nama);

    cout << "Masukkan harga barang: ";
    cin >> harga;

    cout << "Masukkan jumlah beli: ";
    cin >> jumlah;

    float total = harga * jumlah;
    float diskon;

544    if (total > 26.100) {
        diskon = total * 0.10;
    } else {
diskon = 0;
    }

    float total_bayar = total - diskon;

    cout << "\n===== STRUK PEMBELIAN =====" << endl;
    cout << "Nama Barang : " << nama << endl;
    cout << "Harga Satuan : " << harga << endl;
    cout << "Jumlah Beli : " << jumlah << endl;
    cout << "Total Harga : " << total << endl;
    cout << "Diskon : " << diskon << endl;
    cout << "Total Bayar : " << total_bayar << endl;
    cout << "============================" << endl;

    return 0;
}